docs/rackhd/secure_erase.rst

```diff
@@ -141,7 +141,8 @@ Disk Secure Erase Workflow Notes
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Please pay attention to below items if you are using RackHD secure erase function:
 
-* **Use RackHD to manage RAID operation**. RackHD relies on its catalog data for secure erase. If RAID operation is not done via RackHD, RackHD secure erase workflow is not able to recognize drive names given. A suggestion is to re-run discovery for the compute node if you did changed RAID configure not using RackHD
+* **RackHD Secure Erase is not fully tested**. RackHD secure erase is tested on RackHD supported servers with only one LSI RAID controller. Servers with multiple RAID controllers disk array enclosures or non-LSI RAID controllers are not tested.
```

Comment:
No we have no plan to fully test it. RackHD has its own roadmap for hardware support and testing.